### PR TITLE
Move configuration to  a `resume` folder, use YAML

### PR DESCRIPTION
**Pull request by j8r**
This move the configuration presents in the source code directory `src/` to a dedicated configuration folder  `resume/` where we can store our résumé's related data, `data.yml`, and our profile-picture, `id.jpg`.

To enhance the readability of the former `person.js` file, it's now a "yaml" file `data.yml`.
In fact it's still a js one, but more or less *yamlified* for everyone's accessibility on their favorite editor.

To sum up:

This contains **opinionated and breaking changes**, the goal is to be more "beginner friendly"

Discussion is open before merging.

### CODE REVIEW

**Comment by pubkey** [comment]
As you said, we need some discussion here.
I think your yaml-file has a better readability than the previous json-format.
But I'm not sure if that's worthful enough for the breaking change.

File: src/lang/ja.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+const cn = {
+  headings: {
+    contact: '連絡方法',
+    experience: '職務経歴',
+    education: '学歴',
+    skills: 'スキル',
+    about: '自己紹介'
+  }
+};
+export default cn;
```
Comment (j8r) [inline src/lang/ja.js]: `jp` should be used for japanese instead of `cn`

**Comment by salomonelli** [comment]
It is a breaking change but the .yml file is so much easier to read! So I would really like to merge this. Would you mind @j8r, to fix the conflicts?
